contrib/gitian-build.py

```diff
@@ -7,7 +7,7 @@
 
 def setup():
     global args, workdir
-    programs = ['ruby', 'git', 'make', 'wget']
+    programs = ['ruby', 'git', 'make', 'wget', 'curl']
```

Comment:
May I suggest to add the `lsb-release` package which is used here:
https://github.com/bitcoin/bitcoin/blob/76e2cded477bc483ec610212bdadf21fe35292d4/contrib/gitian-build.py#L177
Tested on Debian Buster RC1 NETINST 201900411.

Reply:
How is that going to help? `lsb_release` is called before `setup()`.

Reply:
> `lsb_release` is called before `setup()`.

You are right. The `lsb-release` package is just a prerequisite like the `python3`.